src/components/include/cos_kernel_api.h

```diff
@@ -156,6 +156,12 @@ int cos_introspect(struct cos_compinfo *ci, capid_t cap, unsigned long op);
 
 int cos_sinv(sinvcap_t sinv, word_t arg1, word_t arg2, word_t arg3, word_t arg4);
 
+/*
+ * Calls down to the booter to access necessary resources that are needed for
+ * higher level resource managers
+ */
+void * cos_hypervisor_get_resource(int op, void *arg1, void *arg2, void *arg3);
```

Comment:
Any docs on how to use this function? (also the font is weird here with the `_` and the `g` looking really close together. Funky...)